test/ssltest_old.c

```diff
@@ -1891,7 +1891,7 @@ int doit_localhost(SSL *s_ssl, SSL *c_ssl, int family, long count,
          */
 
         /*
-         * We have non-blocking behaviour throughout this test program, but
+         * We have non-blocking behavior throughout this test program, but
```

Comment:
Hmmm.... So in initialize vs. initialise British(\*) was favoured, but not here?

(\*) It should be noted however that "initialise" lookup at oed.com forwards you  to "initialize" page with *no* mention of "initialise" as [recognized] alternative spelling, while "behavior" forwards to "behaviour" where "behavior" is mentioned as alternative spelling... In other words judging from oed.com "initialise" is probably out of use?